exp-server.js

```diff
@@ -0,0 +1,63 @@
+const mongoose = require('mongoose')
+require("dotenv").config()
+const express = require('express')
+const user = require('./models/User')
+
+
+const app = express()
+
+mongoose.connect(process.env.MONGO_URL)
+
+app.listen(process.env.PORT , () =>
+console.log(`app running om port: ${process.env.PORT}`))
+
+app.get( '/Users',(req, res) =>{
+   user.find({}).exec((err, data) => {
+    if(err){
+        res.send('error')
+    }else{
+        res.json('data')
+        console.log(data)
+    }
+   })
+})
+
+app.post('/Users', (req,res)=> {
+    user.insertOne({
+        name: 'omar',
+        age: 21
```

Comment:
What is the point of always inserting the same user? This endpoint should take the user in its request body.